recipe/conda_build_config.yaml

```diff
@@ -9,6 +9,6 @@ python_impl:
   - cpython
   - cpython
 numpy:
-  - 1.20
-  - 1.20
-  - 1.21.6
+  - 1.20.0
+  - 1.20.0
+  - 1.22.3
```

Comment:
So technically 1.21.6 works, but as per https://github.com/MDAnalysis/mdanalysis/blob/develop/package/pyproject.toml#L17 on windows we actually do 1.22.3, so I thought it'd just be easier to put the lower pin to that for everything